src/Notecard.cpp

```diff
@@ -309,18 +309,24 @@ void Notecard::deleteResponse(J *rsp)
 
 /**************************************************************************/
 /*!
+    @deprecated Do not use.
     @brief  Write a message to the serial debug stream.
     @param    message
               A string to log to the serial debug stream.
 */
 /**************************************************************************/
 void Notecard::logDebug(const char *message)
 {
+    // NoteDebug should be treated as an internal Notecard logging function,
+    // used only by the library itself (note-arduino and note-c) and not its
+    // users.
```

Comment:
Becomes redundant if this messages is moved into the `@deprecated` section of the Doxygen comment.